src/main/java/frc/robot/Controls.java

```diff
@@ -101,11 +102,16 @@ private void configureDrivebaseBindings() {
                   ChassisSpeeds targetSpeeds =
                       new ChassisSpeeds(
                           -driverController.getLeftY()
-                              * MaxSpeed, // Drive forward with negative Y (forward)
+                              * MaxSpeed
+                              * (slowMode ? 0.5 : 1), // Drive forward with negative Y (forward)
```

Comment:
Check `driverController.start().getAsBoolean()` instead of `slowMode`.

Side note (mostly for James)- Controller button triggers are cached, so this doesn't cause unnecessary allocations. We could also store it as a `BooleanSupplier` in the class so that it's guaranteed not reallocated.